Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.8.6-slim-buster as base
+
+RUN apt-get update; apt-get -y install curl; \
+    curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python; \
+    mkdir /app
+WORKDIR /app
+EXPOSE 5000
+
+FROM base as development
+COPY requirements-dev.txt /app
+RUN pip3 install -r requirements-dev.txt; rm requirements-dev.txt
+
+ENTRYPOINT ["python", "/app/app.py"]
+
+FROM base as production
+COPY requirements.txt /app
+RUN pip3 install -r requirements.txt
+COPY static /app/static
+COPY templates /app/templates
+COPY app.py gunicorn_config.py trello_utils.py ViewModel.py wsgi.py /app/
```

Comment:
You should do `.` instead of `/app` since you've set that as the working directory. And in fact you're relying on /app being the current directory when you do the install and run commands

If you have the .dockerignore file set up and there aren't any dev-only files, you could just do `COPY . .`

Reply:
It actually won't let me do ., looks like I have to do ./